cabot/cabotapp/tasks.py

```diff
@@ -22,7 +23,7 @@
 logger = logging.getLogger(__name__)
 
 
-@task(ignore_result=True)
+@task(ignore_result=True, rate_limit=settings.CELERY_RATE_LIMIT)
```

Comment:
We may not need to change `tasks.py` for it. We may be able to add it to settings by using `CELERY_ANNOTATIONS`

http://docs.celeryproject.org/en/latest/userguide/configuration.html#std:setting-task_annotations